pyresample/utils/_proj4.py

```diff
@@ -50,11 +61,14 @@ def proj4_dict_to_str(proj4_dict, sort=False):
         items = sorted(items)
     params = []
     for key, val in items:
-        key = str(key) if key.startswith('+') else '+' + str(key)
-        if key in ['+no_defs', '+no_off', '+no_rot']:
-            param = key
+        if key == 'EPSG':
+            param = 'EPSG:{}'.format(val)
```

Comment:
This assumes we don't have any additional parameters with the epsg code. Should we force this behavior with:

```
params = ['EPSG:{}'.format(val)]
break
```

Or equivalent.

Reply:
Yes, that makes sense. I will implement it.